### ISSUE TITLE
Document a PR to handle role and ARIA a11y tags

### ISSUE THREAD

**Issue by joemasilotti** [joemasilotti/railsdevs.com#459]
Digging through #409 I noticed that there are a lot of a11y tags floating around that might not be 100% accurate. For example, `aria-expanded="false"` is set in the HTML for toggles but is never flipped to `true`.

### A PR addressing this issue should:

- [ ] Audit the entire codebase for _dynamic_ a11y tags that should be updated
- [ ] Document the issues (or fix them!)
- [ ] **BONUS**: Audit the codebase for other a11y/ARIA tags that could be improved

@metamoni, you have first dibs on this one if you'd like to help! No pressure if you don't have the appetite.

**Comment by joemasilotti** [comment]
Awesome - thanks!

**Comment by joemasilotti** [comment]
Thanks for documenting all of those issues this weekend! Are there any more that need to be addresses or can we close this issue for now?

**Comment by metamoni** [comment]
I think there might be a few more SVGs that need to be hidden. Will let you know when we can close this

**Comment by joemasilotti** [comment]
Sounds good – thanks!

**Comment by metamoni** [comment]
We can close this for now, @joemasilotti. If I spot anything else, I'll open a new issue.

**Comment by joemasilotti** [comment]
Sounds great - thanks for documenting all of these!
